lib/provider.js

```diff
@@ -137,6 +157,68 @@ class StorageProvider {
   select(category, query) {
     throw new GSError(errorCodes.NOT_IMPLEMENTED);
   }
+
+  // Get system suffix for given id
+  //   id - <common.Uint64>
+  //
+  // Returns: <common.Uint64>
+  getSystemSuffix(id) {
+    return Uint64.and(id, this.systemBitmask);
+  }
+
+  // Check whether data with given id is stored on this system
+  //   id - <common.Uint64>
+  //
+  // Returns: <boolean>
+  curSystem(id) {
```

Comment:
I'd prefer something more descriptive, i.e.
```suggestion
  isOnCurrentSystem(id) {
```
```suggestion
  isOnCurSystem(id) {
```
And the same for `curServer`.

Reply:
It was moved here from `gs.js`, I didn't change the naming. It's also named in accordance with https://github.com/metarhia/globalstorage/issues/185.

Reply:
I know that it was this way before. I'm suggesting to rename it while we are still at it (and refactoring a lot of things).

Reply:
I think @tshemsedinov should decide on that since he was the one to introduce the naming.

Reply:
I think we can land this as is and I'll open a PR with this change later.